clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2171,7 +2171,7 @@ def GlobalLinkageKind : I32EnumAttr<
     "Linkage type/kind",
     [Global_ExternalLinkage, Global_AvailableExternallyLinkage,
      Global_LinkOnceAnyLinkage, Global_LinkOnceODRLinkage,
-     Global_WeakAnyLinkage, Global_WeakODRLinkage,
+     Global_WeakAnyLinkage, Global_WeakODRLinkage, 
```

Comment:
Trailing space at the end of the line, please remove that. I suggest you configure your IDE to prevent this types of mistakes.